modules/microshift-nw-config-route-admission.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-configuring-route-admission_{context}"]
+= Configuring the route admission policy
+
+By default, {microshift-short} allows routes in multiple namespaces to use the same hostname. To prevent routes in different namespaces from claiming the same hostname, you can configure the value.
+
+.Prerequisites
+
+* You created a {microshift-short} `config.yaml` file.
+* The {oc-first} is installed.
+
+.Procedure
+
+. To prevent routes in different namespaces from claiming the same hostname, update the `namespaceOwnership` field value to `Strict` in the {microshift-short} `config.yaml` file as shown in the following example:
```

Comment:
```suggestion
. To prevent routes in different namespaces from claiming the same hostname, update the `namespaceOwnership` field value to `Strict` in the `config.yaml` file. See the following example:
```

Reply:
Yes, it is required to mention the specific file here.

Reply:
second suggestion is accepted.